README.md

```diff
@@ -62,6 +62,10 @@ The simple idea behind this plugin is to:
 ```lua
 local opts = {
   post_behavior = "echo",  -- or "yank"
+  cat_path = "cat",
+  cat_args = nil,
+  ssh_path = "ssh",
+  ssh_args = nil,
```

Comment:
why not just `cat_cmd` and `ssh_cmd` ? i found this format too verbose... 🤔 
```suggestion
  cat_cmd = "cat -e ", -- can be bat or less
  ssh_cmd = "ssh -B ", -- choose your custom ssh client
```

What do you think ?